### DISCUSSION TITLE
Beatriz Milz - Creating a blog with Quarto in 10 steps

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#33] [General]
# Beatriz Milz - Creating a blog with Quarto in 10 steps

https://beamilz.com/posts/2022-06-05-creating-a-blog-with-quarto/en/

**Comment by DiabbZegpi** [comment]
This post was great! I took this as an inspiration to build my own blog, but I'm having trouble with RSS feed. I've notice that `index.xml` is created inside _site, but then when I use `_site/index.xml` as the path of the RSS feed, the file is not found. **How do you solve the newsletter and the RSS feed in your blog?**

**Comment by trdunsworth** [comment]
Thank you for this. I am building my blog after I finished my PhD and want to use it to promote my work in 911/999/112 and this made it simple to set up. Now comes the hard part. I have to create content for it.
